eureka2-external-server/src/main/java/com/netflix/eureka2/server/EurekaClientProvider.java

```diff
@@ -69,7 +69,6 @@ public Integer call(ServerBootstrap server) {
                 }
         );
         return Eureka.newClientBuilder(discoveryResolver, registrationResolver)
-                .withCodec(config.getCodec())
```

Comment:
We should still be able to do so, the config just provides a way to specify the codec and defaults to avro, it does not force it to avro.